lib/src/view/menu.dart

```diff
@@ -222,6 +223,16 @@ really want to exit without saving.'''
         ..display = 'Import cadnano v2'
         ..key = 'import-cadnano')(),
       DropdownDivider({'key': 'divider-import-cadnano'}),
+      (MenuDropdownItem()
```

Comment:
I'd prefer if we did this the "canonical" React/Redux way of using middleware. Writing to local storage is a side effect (i.e., it modifies something outside of the Redux State), and the traditional way to do side effects is by dispatching an Action and then intercepting the Action in middleware.

So I think we should make a new Action called `ResetLocalStorage` and dispatch it from this menu click. Then write a new middleware (in a new file in the middleware directory) that checks for that action and then executes the code that's written here. Remember to add the new middleware function to the giant list in the file middleware/all_middleware.

The code should look something like

```dart
reset_local_storage_middleware(Store<AppState> store, dynamic action, NextDispatcher next) {
  next(action);
  if (action is actions.ResetLocalStorage) {
    // code you wrote above
  }
}
```